build-system/tasks/check-sourcemaps.js

```diff
@@ -33,6 +33,7 @@ const sourcemapUrlMatcher =
 // Mapping related constants
 const expectedFirstLineFile = 'src/polyfills/array-includes.js';
 const expectedFirstLineCode = 'function includes(value, opt_fromIndex) {';
+const expectedColStart = "'use strict;'".length; // Code begins after an initial 'use strict'.
```

Comment:
Since this is the column at which the first line of real code is supposed to begin, maybe call this `expectedFirstLineColumn` for clarity? (Also, see my suggestion for better documentation.)